doc/markdownlinkconverter/markdownlinkconverter.c

```diff
@@ -0,0 +1,452 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <stdbool.h>
+#include <string.h>
+
+#if defined(WIN32)
+#define FOLDER_DELIMITER '\\'
+#else
+#define FOLDER_DELIMITER '/'
+#endif
+
+/* Slashes or backslashes to the elektra root.
+ * Depends on the place of the executable, example:
+ * /foo/bar/elektra/build/bin/executable
+ * E:\\foo\bar\elektra\build\bin\executable
+ * In both cases the there are 3 slashes or backslashes
```

Comment:
I thinks its better to pass the source directory as arguments and not expect the build directory to be a subdirectory of the source.